modules/atomic-widgets/elements/atomic-svg/atomic-svg.php

```diff
@@ -20,7 +21,6 @@
 
 class Atomic_Svg extends Atomic_Widget_Base {
 	const BASE_STYLE_KEY = 'base';
-	const LINK_BASE_STYLE_KEY = 'link-base';
```

Comment:
Now that we are wrapping the SVG inside a `div` or `a` tag, I don't think we still need this solution.